implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/OtlpHttpMetricsSender.java

```diff
@@ -70,14 +69,12 @@ private String getUserAgentHeader() {
         return userAgent;
     }
 
-    /**
-     * Get the configuration context.
-     * @return A message containing enough information for the log reader to figure out
-     * what configuration details may have contributed to the failure.
-     */
-    private String getConfigurationContext() {
-        // While other values may contribute to failures, these two are most common
-        return "url=" + config.url() + ", resource-attributes=" + config.resourceAttributes();
+    private static class OtlpHttpMetricsSendUnsuccessfulException extends RuntimeException {
```

Comment:
I'm thinking if we should make this more general and public so that other implementations can reuse it and users can get the same exception class for the different implementation.